packages/aws-management/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namePrefix: management-
+
+resources:
+  - ../../modules/aws/eks/cluster
+
+patches:
+  - path: patch-xeks-secretname.yaml
+    target:
+      kind: XEKS
+      name: aws-eks
+  - path: patch-xeks-providerconfig.yaml
```

Comment:
It should be 1 patch instead of 2, as they are the same resource.

Also, do we really need to use the `XRD` here? Shouldn't it be the `claim`? :thinking: